pom.xml

```diff
@@ -327,14 +308,17 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-gpg-plugin</artifactId>
-                        <version>1.5</version>
+                        <version>3.0.1</version>
                         <executions>
                             <execution>
                                 <id>sign-artifacts</id>
                                 <phase>verify</phase>
                                 <goals>
                                     <goal>sign</goal>
                                 </goals>
+                                <configuration>
+                                <keyname>${{ secrets.PUBLISH_SIGNING_KEY_NAME }}</keyname>
```

Comment:
I don't believe this will be interpolated correctly. Moreover, the key identifier/name is not secret - it's necessarily published to centralized keyservers. You can just include it here as plaintext.

`F6D171DC24C6EB30FCAC1E85AEF7D1D58E3C1B9A`

The passphrase and private key data is obviously secret, though.

Reply:
Updated gpg key name as plaintext as recommended .